src/worker.c

```diff
@@ -113,47 +132,45 @@ static CURLMcode init(CURLM *cm, char *method, char *url, int timeout_millisecon
 	(void)pushJsonbValue(&response_headers, WJB_BEGIN_OBJECT, NULL);
 	cdata->response_headers = response_headers;
 	cdata->id = id;
-	cdata->request_headers = request_headers;
-
-	request_headers = curl_slist_append(request_headers, "User-Agent: pg_net/" EXTVERSION);
```

Comment:
request_headers passed to init() now not updated. Previously both cdata->request_headers and request_headers were updated.